posthog/api/test/test_decide.py

```diff
@@ -2624,12 +2624,12 @@ def test_flag_with_behavioural_cohorts(self, *args):
             created_by=self.user,
         )
 
-        with self.assertNumQueries(5):
```

Comment:
Deepeval has updated dependencies, so installing it causes this test to fail. I suspect that the reason is [this line](https://github.com/PostHog/posthog/blob/75e28baf2bc9d1dbba796b3b574218bcbf201c81/posthog/models/feature_flag/flag_matching.py#L476) because snapshots are appended with the two GroupTypeMapping queries that weren't caught previously despite being called on the main branch too.